script/stats.js

```diff
@@ -0,0 +1,56 @@
+// total number of articles and words
+  // <length property>
+// average word length across all posts
+  // <figure out a way to map article and reduce> get into p tags an
+// average word length by author
+
```

Comment:
Try and explain why you have these comments here if you are going to leave them in. Perhaps just a TODO:. Having it at the top here with no explanation is confusing to someone unfamiliar with your code. Would be better at the bottom. Top comments are generally expected to be an expiation of what is in the file, licenses, usage examples, etc.